client/rest-high-level/src/main/java/org/elasticsearch/client/MLRequestConverters.java

```diff
@@ -61,6 +62,26 @@ static Request openJob(OpenJobRequest openJobRequest) throws IOException {
         return request;
     }
 
+    static Request closeJob(CloseJobRequest closeJobRequest) {
+        String endpoint = new EndpointBuilder()
+            .addPathPartAsIs("_xpack")
+            .addPathPartAsIs("ml")
+            .addPathPartAsIs("anomaly_detectors")
+            .addPathPart(closeJobRequest.getCommaDelimitedJobIdString())
+            .addPathPartAsIs("_close")
+            .build();
+        Request request = new Request(HttpPost.METHOD_NAME, endpoint);
+
+        RequestConverters.Params params = new RequestConverters.Params(request);
+        params.putParam("force", Boolean.toString(closeJobRequest.isForce()));
```

Comment:
The need to pass these as params instead of serializing the request revealed a bug: we do not support sending these params as the request body for the close job API. It'd be nice to fix that. Then this converter can follow suit with all the others.

Reply:
👍 Cool, I will create a Github issue for that. It did seem kind of strange as this was the odd API endpoint out :).